CMakeLists.txt

```diff
@@ -698,17 +698,6 @@ else()
   include_directories(BEFORE Externals/curl/include)
 endif()
 
-if(NOT APPLE)
-  check_lib(SOIL "(no .pc for SOIL)" SOIL SOIL/SOIL.h QUIET)
-endif()
-if(SOIL_FOUND)
-  message(STATUS "Using shared SOIL")
```

Comment:
So no shared libpng use for Unix-like systems?

Reply:
We already have the ability to use shared libpng: https://github.com/dolphin-emu/dolphin/blob/9e251206b2616c3e93fb4df93328a22738d9c99f/CMakeLists.txt#L616-L620